besu/src/main/java/org/hyperledger/besu/controller/BesuControllerBuilder.java

```diff
@@ -335,7 +342,7 @@ protected SubProtocolConfiguration createSubProtocolConfiguration(
     return new SubProtocolConfiguration().withSubProtocol(EthProtocol.get(), ethProtocolManager);
   }
 
-  protected final void addShutdownAction(final Runnable action) {
+  final void addShutdownAction(final Runnable action) {
```

Comment:
Curious - why change to package-private here?

Reply:
IntelliJ detected they were only used in package private use cases.  Principal of least privilege.